e2e/utils/port_config.yaml

```diff
@@ -3,4 +3,4 @@ kind: ConfigMap
 metadata:
   name: config-port-config
 data:
-  sample-service.PORT: "8888"
+  sample-service.PORT: "9001"
```

Comment:
Should port be an environment variable that we can change as needed?

Reply:
There will be more than one microservices eventually, hence env vars will not work. Let's upgrade this later.